insertion_sort.py

```diff
@@ -0,0 +1,42 @@
+def insort(unlist):
+    """Insertion sort a list
+
+    Implementation follows after gif here
+    https://en.wikipedia.org/wiki/Insertion_sort"""
+    slist = unlist[:]
+    for i in range(1, len(slist)):
+        j = i - 1
+        while j >= 0:
+            if slist[i] < slist[j]:
+                slist[i], slist[j] = slist[j], slist[i]
+                j -= 1
+                i -= 1
+            else:
+                break
+    return slist
+
+
+if __name__ == "__main__":
+    """Test time performance for best and worst cases"""
```

Comment:
The assignment asks that you "demonstrates the performance characteristics of this sort over a variety of lengths of input in both the best and worst-case scenarios." There is only one input length here.